fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java

```diff
@@ -189,6 +189,7 @@ public CreateTableStmt(boolean ifNotExists, boolean isExternal, TableName tableN
         this.partitionDesc = partitionDesc;
         this.distributionDesc = distributionDesc;
         this.properties = properties;
+        PropertyAnalyzer.getInstance().rewriteForceProperties(this.properties);
```

Comment:
better to rewrite them in Nereids CreateTableCommand

Reply:
Try to rewrite properties in all possible path.